### PR TITLE
feature(TG-367): add corect links to device statistic from list

### PR DESCRIPTION
**Pull request by velmyk**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/200" target="_blank">ReviewNinja

File: src/app/shared/devices/device-list/device-list.template.html [hunk 1]
```diff
@@ -12,20 +21,20 @@
         </thead><!-- sensors-list-table__head -->
         <tbody class="sensors-list-table__body">
             <tr *ngFor="let item of deviceList"
-                [routerLink]="['./', item._id]"
+                [routerLink]="editOnItemClick ? ['./', item._id] : []"
```
Comment (ashilay) [inline src/app/shared/devices/device-list/device-list.template.html]: It is better to move such logic into controller and cover with unit tests

Comment (piton13) [inline src/app/shared/devices/device-list/device-list.template.html]: Will it stay on the same state when editOnItemClick equals false?

File: src/app/shared/devices/device-list/device-list.component.test.ts [hunk 2]
```diff
@@ -53,9 +70,38 @@ describe('device-list', () => {
         });
     });
 
+    describe('#goToDeviceStatistic', () => {
+        let event;
+        const mqttId = Symbol('device id to show statistic');
+        const statisticLink = 'path to statistic';
+
+        beforeEach(() => {
+            event = {
```
Comment (valakar) [inline src/app/shared/devices/device-list/device-list.component.test.ts:79]: You can mock it in parent beforeEach once

Comment (velmyk) [inline src/app/shared/devices/device-list/device-list.component.test.ts:79]: Thank you for mention, but it not used in every test, so i'll better leave it in particular tests (beforeEach's).

File: src/app/shared/devices/device-list/device-list.template.html (hunk 1 above)
Comment (velmyk) [inline src/app/shared/devices/device-list/device-list.template.html]: !resolved

File: src/app/shared/devices/device-list/device-list.component.ts [hunk 3]
```diff
@@ -17,8 +18,9 @@ const headersForDisplay = [
 })
 export class DeviceListComponent {
     @Input() deviceList: Array<Device>;
+    @Input() editOnItemClick: boolean = true;
     @Input() showDeleteButton: boolean = true;
-    @Input() showStatisticLink: boolean = true;
+    @Input() statisticLink: string|boolean = false;
```
Comment (piton13) [inline src/app/shared/devices/device-list/device-list.component.ts]: !fix
Will be better to use only one type of variable and in *ngIf just check if length of string more than 0.

Comment (velmyk) [inline src/app/shared/devices/device-list/device-list.component.ts]: I think current version i more readable

Comment (piton13) [inline src/app/shared/devices/device-list/device-list.component.ts]: I don't know if I can assign to the property true or false value. It seems that 'false' value uses only for initial state. And 'true' - never.

Comment (velmyk) [inline src/app/shared/devices/device-list/device-list.component.ts]: !fixed

**Comment by velmyk** [comment]
Last approve from myself because this PR already had a lot of stars which despaired after minor updates. Want to merge before go to bed)

**Pull request merged**